config/settings/test.yml

```diff
@@ -1,3 +1,6 @@
+features:
+  pentest_partners_access_enabled: true
```

Comment:
The problem with setting this here is that https://github.com/alphagov/forms-admin/pull/712/files#diff-a66e72dce6f86faae2ba62aa5dd129e12ced85afd75419bf4668b71d0344236aR9 will never get exercised during the tests. I tested this locally by putting a breakpoint in the else block and not once did it pause the tests from running